src/utils/queries.js

```diff
@@ -0,0 +1,38 @@
+export const createAgentTable = `
+DROP TABLE IF EXISTS agents;
+CREATE TABLE "agents" (
+  "id" serial primary key,
+  "first_name" varchar(255) NOT NULL,
+  "last_name" varchar(255) NOT NULL,
+  "email" varchar UNIQUE NOT NULL,
+  "password" varchar NOT NULL,
+  "phone_no" varchar NOT NULL
```

Comment:
you are validating the `phone_no` value as a number but you are saving it as a string, why sir? 👀